tools/ola_trigger/Context.cpp

```diff
@@ -66,12 +72,26 @@ void Context::Update(const string &name, const string &value) {
 
 
 /**
+ * @brief Set the config file variable
+ */
+void Context::SetConfigFile(const string &config_file) {
+  m_variables[CONFIG_FILE_VARIABLE] = config_file;
```

Comment:
I switched this from creating a new string object, given the Update function above didn't seem to bother @nomis52 .